integration-tests/e2e/resources/managed/kustomization.yaml

```diff
@@ -1,11 +1,12 @@
 ---
-kind: Kustomization
 apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
 
 namespace: ${managed_namespace}
+
 resources:
+  - ../../../shared/resources/managed
```

Comment:
It can't work well if the same yaml files exist in the current folder and shared folder.